packages/loot-core/src/server/budget/goaltemplates.ts

```diff
@@ -563,10 +569,15 @@ async function applyCategoryTemplate(
           amountCond.value = monthlyTarget;
         }
 
-        if (template.full === true) {
-          if (num_months === 1) {
+        if (template.full === true || isReflectBudget()) {
+          if (num_months === 0) {
```

Comment:
^ same here — was this change intentional?

Reply:
This was intentional.  The calculation wasn't happening.  I think it broke when we started using the monthUtils package to calculate dates.